main/config/settings/.defaults.yml

```diff
@@ -201,4 +202,7 @@ egressNotificationTopicName: ${self:custom.settings.namespace}-EgressTopic
 # If you toggle the AppStream feature from true to false and redeploy the whole solution, the backend stack deployment shold error out with following message:
 # Error validating existing stack policy: Unknown logical id 'LogicalResourceId/AppStream*' in statement {} - stack policies can only be applied to logical ids referenced in the template
 # Stack policies are applied here: addons/addon-stack-policy/packages/stack-policy/lib/steps/update-cfn-stack-policy.js
-isAppStreamEnabled: false
\ No newline at end of file
+isAppStreamEnabled: false
+
+# The name of the S3 bucket used to store uploaded study data
+studyDataBucketName: ${self:custom.settings.globalNamespace}-studydata
```

Comment:
Do we need `deploymentBucketName` in other `.defaults.yml` files if we have it added here. I wondered that because `deploymentBucketName` is just initialized here once but referenced in cloudformation.yml of post-deployment and backend both

Reply:
Doesn't the fact that it is in the main `.defaults.yml` mean that all SDCs will have this value _unless_ it is overwritten in another SDC settings file? This is the only place that it is a parameter so this value will be used by all SDCs I thought.